mime/cupsfilters.types

```diff
@@ -91,6 +91,9 @@ image/x-xbitmap			xbm string(0,"#define");
 image/x-xpixmap			xpm string(3,"XPM")
 image/x-xwindowdump		xwd string(4,<00000007>)
 image/urf			urf string(0,UNIRAST<00>)
+image/jxl       		jxl string(0,"<0000000C4A584C20>")
+image/jxl			jxl string(0,"<FF0A>")
+
```

Comment:
This should work in one line:
```
image/jxl       		jxl string(0,"<0000000C4A584C20>") string(0,"<FF0A>")
```
Each space-separated expression is logically ORed.